Makefile

```diff
@@ -51,7 +51,9 @@ update:
 verify:
 	@which go-bindata 2> /dev/null >&1 || { echo "go-bindata must be installed to verify generated code";  exit 1; }
 	@which golangci-lint 2> /dev/null >&1 || { echo "golangci-lint must be installed to lint code";  exit 1; }
+	@which gosec 2> /dev/null >&1 || { echo "gosec must be installed to lint code";  exit 1; }
 	golangci-lint run
+	gosec -severity high --confidence medium -exclude G204 -quiet ./...
```

Comment:
let's add a comment pointing to https://github.com/golangci/golangci-lint/issues/597 to make sure we get rid of this once golangci-lint updates gosec

Reply:
Done.

Reply:
uhm, did you push? 😄

Reply:
lol, yep. NOW I pushed :smiley: